common/templates/options.php

```diff
@@ -77,6 +77,20 @@
                     <?php if (!$plugin->woocommerce): ?>
                     <p class="description"><?php _e('Install and activate WooCommerce to use this functionality.', 'webwinkelkeur'); ?></p>
                     <?php endif; ?>
+                    <fieldset>
+                        <br>
+                        <label for="<?= $plugin->getOptionName('order_status'); ?>"> <?php _e('Select order status', 'webwinkelkeur'); ?>
+                            <select name="<?= $plugin->getOptionName('order_status'); ?>">
+                                <?php foreach (wc_get_order_statuses() as $key => $label): ?>
+                                    <option value="<?= $key; ?>" <?= ($key == $config['order_status']) ? ' selected="selected"' : ''; ?>><?= _e($label, 'webwinkelkeur'); ?>
```

Comment:
* No need to write `selected="selected"`, simply `selected` is plenty.
* There's no need to put parentheses around the condition in the ternary operator.
* You already have a space before `<?`, there's no need to put another one inside the string.
* Translating variables doesn't work, the translation tool won't pick up those values from the source code. Besides, the status labels are already translated:

https://github.com/woocommerce/woocommerce/blob/8db66f705f5906ff8e6f7f105c095fb71774dc40/includes/wc-order-functions.php#L93-L104